core/deployment/src/main/java/io/quarkus/deployment/util/DockerWorking.java

```diff
@@ -11,8 +11,6 @@
 
 import org.jboss.logging.Logger;
 
-import io.quarkus.deployment.util.ExecUtil;
-
 public class DockerWorking implements BooleanSupplier {
```

Comment:
Can you rename this class to `IsDockerRunning` to match the existing `BooleanSupplier` classes?

Reply:
Also move it to the `io.quarkus.deployment` package